cfgov/ask_cfpb/scripts/migrate_knowledgebase.py

```diff
@@ -142,9 +142,49 @@ def fix_tips(answer_text):
 
 
 def get_or_create_landing_pages():
-    """Create English and Spanish landing pages"""
+    """
+    Create two Spanish and one English landing pages.
+
+    Spanish gets two parents:
+    - `/es/`, as a placeholder for a future Spanish home page in Wagtail
+    - `/obtener-respuestas/`, the landing page for Spanish ask-cfpb
+    """
+
     from v1.models import CFGOVPage
-    parent = CFGOVPage.objects.get(slug='cfgov').specific
+    root = CFGOVPage.objects.get(slug='cfgov').specific
+
+    def get_or_create_es_parent_page():
+        """Create the Spanish /es/ parent placeholder page"""
+        text_intro_stream_value = [
+            {'type': 'text_introduction',
+             'value': {
+                 'heading': 'Placeholder for future Spanish home page',
+                 'links': []}}]
+        # this unserved archived page is squatting on the /es/ slug
+        dinero = CFGOVPage.objects.get(id=51)
```

Comment:
Actually, _this_ unpublished page in CFGov is squatting on `/es` for the purposes of being able to serve [MAYG Spanish](https://www.consumerfinance.gov/es/el-dinero-mientras-creces/) and [Tax Time Saving Spanish](https://www.consumerfinance.gov/es/opciones-de-ahorro-con-el-reembolso-de-sus-taxes/): [Prod Wagtail domain]/admin/pages/3637/